pages/news/2020-08-XX-new-in-2-3-deck-clone.html

```diff
@@ -0,0 +1,53 @@
+title: New In 2.3: Deck Cloning
+author: Matthew Nicholson
+date: 2020-08-XX 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+
+Deck cloning is a highly anticipated feature added in Mixxx 2.3. In short, it does what it says on the tin. When activated, the currently loaded song, playback state, pitch adjustment, and any playing loop will be cloned from one deck to another. This opens up a number of creative mixing possibilities, some of which we will explore below. Note: in some other DJ software, this feature is known as "instant doubles".
+
+# How Do I Use Deck Cloning?
+
+Deck clone can be triggered several different ways.
+
+- Double press the load button on your controller, mixer, or keyboard (this can be disabled in the deck options settings).
+- Invoke the `CloneFromDeck` control in a controller mapping.
+- Drag and drop a track from one deck to another in the UI.
+
+Any of these actions will result in the currently loaded track being loaded on the target deck at the current play position and pitch setting. If the track is playing on the source deck, it will play from the same exact position on the target deck.
+
```

Comment:
I would be cool if you could add a little GIF here that demonstrates deck cloning on a playing deck by drag and drop.